.env.example

```diff
@@ -8,4 +8,4 @@ TONAPIIO_MAINNET_URL=https://tonapiio.mytonwallet.org
 TONAPIIO_TESTNET_URL=https://tonapiio-testnet.mytonwallet.org
 PROXY_HOSTS="tonproxy.io:38080 tonproxy.io:38081 tonproxy.io:38082"
 BRILLIANT_API_BASE_URL=
-STAKING_POOLS=
+STAKING_POOLS=
```

Comment:
Can you roll back the changes to this file completely?

Reply:
No I created a new file that I took from your master branch.

Reply:
Can you add a line break on the last line please?

Reply:
Okey